digits/extensions/view/interface.py

```diff
@@ -49,6 +49,32 @@ def get_header_template(self):
         """
         return None, None
 
+    def get_ng_templates(self):
```

Comment:
I agree, @gheinrich. I should at least rename it to something library agnostic, like `app_begin_html` and `app_end_html` or something to that effect. I didn't insert the app header html into the existing header because the header is wrapped in a `div`, and the app code needs to have the interface views as child elements for scope purposes.

I had added a footer, but it was not used and subsequently removed.